.pre-commit-config.yaml

```diff
@@ -9,24 +8,18 @@ repos:
         language: system
         files: '^webui/react/src/'
         types_or: [javascript, jsx, ts, tsx]
-        stages:
-          - commit
```

Comment:
Why these changes?

Reply:
hmm these are not supposed to be here :eye I'll take them out

Reply:
done. I had split these up to https://github.com/determined-ai/determined/pull/6034 not sure how it ended up being reverted here.